app/models/cloud_tenant.rb

```diff
@@ -140,6 +140,12 @@ def update_source_tenant_associations_for_vms_and_templates
     end
   end
 
+  def update_source_tenant(tenant_params)
+    _log.info("CloudTenant #{name} has tenant #{source_tenant.name}")
+    _log.info("Updating Tenant #{source_tenant.name} with parameters: #{tenant_params.inspect}")
```

Comment:
Please combine these two log lines.  If you add the CloudTenant name to this second log line, the first becomes redundant.